turkiye/turkiye_earthquake_legitimate_iocs.csv.csv

```diff
@@ -0,0 +1,25 @@
+﻿type,indicator,classification,references
+domain,yikilanbinalar.com.tr,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,enkazbildirim.org,legitimate,https://github.com/SOCRadar/TurkeyEarthquakeIntel/blob/main/TrustedList.txt
+domain,howtohelpturkey-syriaearthquakevictims.org,legitimate,

```

Comment:
why do we know this is legit?

Reply:
"Domain contains information about how can people can support the people of Turkey and Syria which were hit by the earthquake on February 2023. It contains links to legitimate funding initiatives."

Reply:
ok